modules/extensions/eprocess/publisher/eprocess/apis/processes.jag

```diff
@@ -0,0 +1,34 @@
+<%
+
+require('/modules/publisher.js').exec(function(ctx) {
+	var log=new Log();
+	var req=ctx.request;
+	var res=ctx.response;
+
+	var q=req.getParameter('q');
+	var type=req.getParameter('type');
+	var asset=require('rxt').asset;
+	var server=require('store').server;
+
+	if(!q){
```

Comment:
This needs to be switched so that logic is in the if block. Will be done after pull request merge